src/main/java/org/apache/commons/lang3/reflect/MethodUtils.java

```diff
@@ -142,21 +193,57 @@ public static Object invokeMethod(final Object object, final String methodName,
      * @throws InvocationTargetException wraps an exception thrown by the method invoked
      * @throws IllegalAccessException if the requested method is not accessible via reflection
      */
-    public static Object invokeMethod(final Object object, final String methodName,
+    public static Object invokeMethod(final Object object, final boolean forceAccess, final String methodName,
             Object[] args, Class<?>[] parameterTypes)
             throws NoSuchMethodException, IllegalAccessException,
             InvocationTargetException {
         parameterTypes = ArrayUtils.nullToEmpty(parameterTypes);
         args = ArrayUtils.nullToEmpty(args);
-        final Method method = getMatchingAccessibleMethod(object.getClass(),
-                methodName, parameterTypes);
+        
+        final String messagePrefix;
+        final Method method;
+        if (forceAccess) {
+            messagePrefix = "No such method: ";
+            method = getMatchingMethod(object.getClass(), methodName, parameterTypes);
+            if (method != null) {
+                method.setAccessible(true);
```

Comment:
Agreed -- I didn't realize that the issue was out there and wanted to be consistent with FieldUtils.   I'll make this change.   Thanks for making me aware of it.